db/run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+name='singleMySql'
```

Comment:
It's database name or just a program name?

Reply:
Name of container